pkg/cpio/cpio.go

```diff
@@ -81,6 +81,27 @@ func NewDedupWriter(rw RecordWriter) RecordWriter {
 	}
 }
 
+// Passthrough reads in a cpio from a root and writes it to a destination root.
+// This avoids copying the entire partition which is much larger than the cpio.
+func Passthrough(r RecordReader, w RecordWriter) error {
+	for {
+		rec, err := r.ReadRecord()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
```

Comment:
Should we print out the record here with %q as well?

Reply:
Maybe. Would it clutter up the terminal too much when called?

Reply:
it's an error. It will only print an error once, and then it's done. I think it's ok

Reply:
I meant "include the error in the fmt string"

Reply:
rewrite as
func PassThrough(r, w) error {
return Concat(r, w, nil)
}

passthrough is a well known  cpio term so this will make sense to people. I never noticed Concat.

Reply:
What should I do about WritingTrailer? I could rewrite it to call Concat then write the trailer

Reply:
yeah, let's do that. We may later find we want to make writing the trailer optional but .. let's do that.